File: codingTest/baekjoon/javascript/10989.js
var rl = require("readline").createInterface({ input: process.stdin, output: process.stdout });
var flag = true;
var arr = new Array(10001).fill(0);
rl.on("line", line => {
  if (flag) flag = false;
  else arr[line] += 1;
}).on("close", () => {
  for (let i = 0; i < 10001; i++) if (arr[i] != 0) while (arr[i]--) console.log(i);
});

File: codingTest/baekjoon/python/10989.py
import sys
n = int(sys.stdin.readline())
arr = [0] * 10001
for _ in range(n):
    arr[int(sys.stdin.readline())] += 1
for i in range(10001):
    if arr[i] != 0:
        for _ in range(arr[i]):
            print(i)